mdn/developer-portal
# Add community role and remove job titles from cards

This changeset removes job titles from people cards and alters the supported roles slightly. It also replaces the job title within the person's profile header with their role (to match the icon).

These roles are _not_ permission-related. They are just displayed on person profile pages.

## Key changes

- Merge "guest" and "contributor" roles into a new "community" role.
- Update references to the old roles across templates.
- Show a person's role instead of their job title within cards and the profile header.

## How to test

- View the people directory page and observe how roles are now shown within cards.
- View a person's profile page and observe how their role is shown within the header.
- Change a person's role to "community" and observe the icon change throughout.
